src/main/java/jenkins/plugins/git/AbstractGitSCMSource.java

```diff
@@ -243,22 +310,52 @@ protected void retrieve(@CheckForNull final SCMSourceCriteria criteria,
             @Override
             public Void run(GitClient client, String remoteName) throws IOException, InterruptedException {
                 final Repository repository = client.getRepository();
-                listener.getLogger().println("Getting remote branches...");
+                listener.getLogger().println("Getting remote branches/tags...");
```

Comment:
Probably have the context information to be able to report whether it is tags or branches or both, would be good to use that rather than a generic message as it helps users debug